components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -0,0 +1,78 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-by-user",
+  name: "New comments by user",
```

Comment:
By convention we like to capitalize all words in the source name, so this should read "New Comments By User"

Reply:
Would you mind changing the other sources to match that format, as well?

Reply:
Well were changed to that format, yes.